vignettes/miaTime.Rmd

```diff
@@ -274,6 +274,25 @@ colData(se)$time_from_endpoint <- colData(se)[, "time"] -
 
 ```
 
+Sample dissimilarity between consecutive time steps(step size n >= 1) within
+a group(subject, age, reaction chamber, etc.) can be calculated by  	`getTimeDivergence`.
+
+```{r}
+library(miaTime)
+data("hitchip1006")
+se <- hitchip1006
+
+se <- se[, colData(hitchip1006)$subject %in% c("900", "934", "843", "875")]
```

Comment:
why hitchip1006 used on this line because we already defined on line 283 that se <- hitchip1006